capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
+end
```

Comment:
- [ ] Instead of calling super I would suggest you do call the method

```ruby
    def correct_name
      @nameable.correct_name.capitalize
    end

```